app/bundles/CoreBundle/Tests/Functional/RememberMe/RememberMeTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Mautic\CoreBundle\Tests\Functional;
```

Comment:
```suggestion
namespace Mautic\CoreBundle\Tests\Functional\Config;
```
Can we rename the new folder to Config so more tests regarding config issues could be add in it?